### ISSUE TITLE
Abbreviated names are missing from Instruments Panel for some templates

### ISSUE THREAD

**Issue by HemantAntony** [musescore/MuseScore#12933]
**Describe the bug**
The abbreviated names are missing in Instruments Panel

**To Reproduce**
1. Open the String Quartet Template (Other chamber music templates can also be used)
2. Go to Instruments Panel
3. Click on Viola Settings Button
4. Abbreviated name is missing

**Expected behavior**
Abbreviated names should be displayed

**Screenshots**
![image](https://user-images.githubusercontent.com/88136054/185632074-d0ea5f23-1f4c-470b-9255-e8d67a4189d6.png)

**Platform information**
 - OS: Ubuntu 22.04 LTS

**Additional context**
This problem occurs for some templates
@Tantacrul FYI

**Comment by HemantAntony** [comment]
Also, the Voice + Piano template is missing names for its instruments too

**Comment by cbjeukendrup** [comment]
This may be by design. For small standard ensembles like a string quartet, the abbreviated names don't add much clarity; it's already clear enough which staff belongs to which instrument. And for the Voice+Piano combination, it's so trivial, that the even the long names at the start would be redundant. 

Anyway, I think @oktophonie can best decide about this.

**Comment by MarcSabatella** [comment]
Indeed, this definitely was by design originally (e.g., in MuseScore 3, probably longer than that).  Whether the design should be revised or not is another matter.  But I would vote no, since most small chamber scores I see do not show abbreviated names.

**Comment by oktophonie** [comment]
I can't help but feel it would be better to control the visibility of instrument labels via style settings, something roughly like:

- On first system of score/section
    - [ ] long name
    - [ ] short name
    - [ ] no name

- On subsequent systems
    - [ ] long name
    - [ ] short name
    - [ ] no name
    
The long/short labels are, after all, part of the instrument definition; we just want to control how/whether they are shown, in any given context. (We already have the options for hiding them if there's just a single instrument, which expedites parts.) It's tedious, after all, to create a small chamber score and then have to delete the short names for each instrument just to hide them.

**Comment by oktophonie** [comment]
Was addressed by way of https://github.com/musescore/MuseScore/issues/18289 / https://github.com/musescore/MuseScore/pull/22550 (in 4.4)
